themes/godotengine/pages/fbx-import.htm

```diff
@@ -0,0 +1,106 @@
+is_hidden = 0
+==
+{##}
+{% put title %} FBX support in Godot {% endput %}
+
+<p>
+  <em>The information on this page only pertains to Godot 4.0 and later. Godot 3.x does not have support for using fbx2gltf directly, but you can use it as a command line tool to convert scenes before placing the converted glTF file in the Godot project folder.</em>
+</p>
+<hr>
+<p>
+  <a href="https://en.wikipedia.org/wiki/FBX">FBX</a> is a popular proprietary
+  3D interchange format. It is often used in game engines to allow importing 3D
+  scenes created using software such as Blender, Maya or 3DS Max.
+</p>
+<p>
+  Godot does not include an FBX importer in its editor code, as doing so would
+  require linking against a proprietary SDK. Relying on reverse engineering was
+  <a href="https://godotengine.org/article/fbx-importer-rewritten-for-godot-3-2-4">attempted in the past</a>,
+  but it turned out to be too unreliable.
+</p>
+<p>
+  However, thanks to a tool known as FBX2glTF, <strong>it is still possible to import
+  FBX scenes to Godot</strong> in a transparent manner. But first…
+</p>
+
+<h3 class="title">Consider using glTF 2.0 instead</h3>
+<p>
+  <a href="https://en.wikipedia.org/wiki/GlTF">glTF</a> is an open 3D
+  interchange format. Unlike FBX, it has many open source implementations
+  available and is supported out of the box in Godot.
+</p>
+<p>
+  We recommend using glTF over FBX for several reasons:
+  <ul>
+    <li>glTF is better supported in open source 3D software such as Blender.</li>
+    <li>
+      glTF import is integrated in the Godot editor. This way, team members
+      working on your project don't need to install external tools before they
+      can start working on the project.
+    </li>
+    <li>
+      If you use headless exporting (continuous integration) for your project,
+      you don't need to install external tools before you can export the
+      project.
+    </li>
+    <li>
+      If you use the Android or HTML5 editor, you can import glTF scenes, but
+      not FBX scenes as these platforms don't allow running arbitrary external
+      executables.
+    </li>
+    <li>
+      If you use a device on a non-x86 architecture, you can import glTF scenes,
+      but not FBX scenes as the Autodesk FBX SDK is only available for x86
+      architectures.
+    </li>
+    <li>
+      glTF is an open standard. Unlike FBX, it doesn't force other software to
+      link against a proprietary SDK to reliably import 3D scenes.
+    </li>
+  </ul>
+<p>
+  See <a href="https://godotengine.org/article/we-should-all-use-gltf-20-export-3d-assets-game-engines">this article</a> for more information.
+</p>
+
+<h3 class="title"">Download FBX2glTF</h3>
+<p><strong>
+  By downloading FBX2glTF, you accept the
+  <a href="https://nightly.link/godotengine/FBX2glTF/workflows/build.yaml/master/thirdparty-license.rtf.zip">Autodesk LICENSE AND SERVICES AGREEMENT for Autodesk FBX SDK 2020</a>
```

Comment:
GitHub has a way to link downloads in the latest GitHub Release, so we should use that instead of nightly.link.
This also gives us the .rtf file directly instead of a zip.
```suggestion
  <a href="https://github.com/godotengine/FBX2glTF/releases/latest/download/FBX-SDK-License.rtf">Autodesk LICENSE AND SERVICES AGREEMENT for Autodesk FBX SDK 2020</a>
```